luis-hurtado/tarea-1/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Tarea-1</title>
+</head>
+<body>
+    <main class="container" >
+        <article>
```

Comment:
@luishurtado-globant , this should be a header

Reply:
Hi, the main tag or the article?

Reply:
@luishurtado-globant , the article